build/Common.props

```
@@ -37,6 +37,7 @@
     <OpenTelemetryCoreLatestVersion>[1.4.0,2.0)</OpenTelemetryCoreLatestVersion>
     <OpenTelemetryCoreLatestPrereleaseVersion>[1.5.0-alpha.1]</OpenTelemetryCoreLatestPrereleaseVersion>
     <StackExchangeRedisPkgVer>[2.1.58,3.0)</StackExchangeRedisPkgVer>
+    <CassandraCSharpDriverPkgVer>[3.16.0,3.19.1)</CassandraCSharpDriverPkgVer>
```

If I understand correctly, This instrumentation will work with packaged from 3.16.0 to 3.19.0. 3.19.1 will rise warnings/errors by nuget package.

Consider: 
```suggestion
    <CassandraCSharpDriverPkgVer>[3.16.0,4.0)</CassandraCSharpDriverPkgVer>
```